hadoop-hdds/common/src/main/resources/hdds-version-info.properties

```diff
@@ -25,4 +25,5 @@ url=${version-info.scm.uri}
 srcChecksum=${version-info.source.md5}
 hadoopProtoc2Version=${proto2.hadooprpc.protobuf.version}
 hadoopProtoc3Version=${proto3.hadooprpc.protobuf.version}
-grpcProtocVersion=${grpc.protobuf-compile.version}
\ No newline at end of file
+grpcProtocVersion=${grpc.protobuf-compile.version}
+platform=${os.detected.classifier}
```

Comment:
Looks like this value is defined in compile time.

Reply:
Yes, the value is from the below platform.